server/Pipfile

```diff
@@ -0,0 +1,19 @@
+[[source]]
+
```

Comment:
Are these empty lines necessary? Or is it me not used to Python conventions? :)

Reply:
I think that as long as the code is consistent then the formatting is not necessary, not only in Python case.